config/config.go

```diff
@@ -17,13 +17,15 @@ import (
 
 // Configuration specifies the static application config.
 type Configuration struct {
-	ExternalURL string     `mapstructure:"external_url"`
-	Host        string     `mapstructure:"host"`
-	Port        int        `mapstructure:"port"`
-	Client      HTTPClient `mapstructure:"http_client"`
-	CacheClient HTTPClient `mapstructure:"http_client_cache"`
-	AdminPort   int        `mapstructure:"admin_port"`
-	EnableGzip  bool       `mapstructure:"enable_gzip"`
+	ExternalURL  string     `mapstructure:"external_url"`
+	Host         string     `mapstructure:"host"`
+	Port         int        `mapstructure:"port"`
+	EnableSocket bool       `mapstructure:"enable_socket"`
+	Socket       string     `mapstructure:"socket"`
```

Comment:
Thank you for adding the enable flag. To be a bit nitpicky here, I think the name is ambiguous. Consider renaming to "UnixSocketEnabled" and UnixSocketName", or you can create a sub-object like:

```
UnixSocket  UnixSocketConfig       `mapstructure:"unix_socket"`

type UnixSocketConfig struct {
  Enabled   bool       `mapstructure:"enabled"`
  Name.      string.    `mapstructure:"name"`
}
```